hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -81,6 +81,12 @@ public record ContractsConfig(
         @ConfigProperty(value = "systemContract.accountService.isAuthorizedRawEnabled", defaultValue = "true")
                 @NetworkProperty
                 boolean systemContractAccountServiceIsAuthorizedRawEnabled,
+        @ConfigProperty(value = "systemContract.accountService.isAuthorizedEnabled", defaultValue = "true")
+                @NetworkProperty
+                boolean systemContractAccountServiceIsAuthorizedEnabled,
+        @ConfigProperty(value = "systemContract.metadataKeyAndFieldSupport.enabled", defaultValue = "false")
```

Comment:
I know that the flag is disabled, but we should not include it for this release

Reply:
I picked this up inadvertently not realizing it wasn't supposed to go into 0.56.  There are _no usages_ of this method.  Will leave it in as it is harmless (for expediency in getting this PR merged to release/0.56).